doc/install_windows_manually.txt

```diff
@@ -1,5 +1,5 @@
 --------------------------------------
-Firebird 2.1 manual installation notes
+Firebird 4.0 manual installation notes
```

Comment:
Here and later - I do not think it's worth specifying particular version here. To avoid misunderstanding or need to change this version every new release better say just:
Firebird manual installation notes
Same for other places where version 4 is mentioned.

Reply:
I have done so.